main.tf

```
@@ -0,0 +1,15 @@
+terraform {
+  required_version = "1.23.0"
```

1.23.0 isn't a thing as far as i can tell
```suggestion
  required_version = "1.1.2"
```